src/courses/templates/courses/block_list.html

```diff
@@ -26,11 +27,15 @@
   <tbody>
     {% for object in object_list %}
     <tr {% if not object.active %} class="text-muted" {% endif %}>
-      <td>{{ object.name }}</td>
+      <td>{{ object.name }}</a></td>
```

Comment:
Random closing `a` tag?

Reply:
Random closing </a> tag removed. That was an uncaught remnant of when the list view was accessed through a hyperlinked name instead of a button